cas/src/main/java/org/springframework/security/cas/web/authentication/DefaultServiceAuthenticationDetails.java

```diff
@@ -70,14 +71,13 @@ public String getServiceUrl() {
 
 	@Override
 	public boolean equals(Object obj) {
```

Comment:
Generally, I'm not in favor of these changes as the potential benefits are a matter of opinion